src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -433,6 +433,12 @@ function createFiberFromElementType(
 
 exports.createFiberFromElementType = createFiberFromElementType;
 
+exports.createFiberFromHostInstanceForDeletion = function(): Fiber {
+  const fiber = createFiber(HostComponent, null, NoContext);
+  fiber.type = 'DELETED';
```

Comment:
`createFiber` is not exposed at all. Makes it easier to add or change the default inputs here.

`stateNode` is not passed when other `HostComponent` Fibers are created so I figured we'd be consistent there.